vignettes/checks/plausibleAfterBirth.Rmd

```diff
@@ -0,0 +1,94 @@
+---
+title: "plausibleAfterBirth"
+author: "Maxim Moinat, Katy Sadowski"
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    number_sections: yes
+    toc: yes
+---
+
+## Summary
+
+**Level**: Field check\
+**Context**: Verification\
+**Category**: Plausibility\
+**Subcategory**: Temporal\
+**Severity**: Characteristic
+
+## Description
+The number and percent of records with a date value in the **cdmFieldName** field of the **cdmTableName** table that occurs prior to birth.
+
+## Definition
+This check verifies that events happen after birth. This check is only run on fields where the **PLAUSIBLE_AFTER_BIRTH** parameter is set to **Yes**. The birthdate is taken from the `person table`, either the `birth_datetime` or composed from `year_of_birth`, `month_of_birth`, `day_of_birth` (taking 1st month/1st day if missing).
+
+- *Numerator*: The number of records with a non-null date value that happen prior to birth.
+- *Denominator*: The total number of records in the table with a non-null date value.
+- *Related CDM Convention(s)*: -Not linked to a convention-
+- *CDM Fields/Tables*: By default, this check runs on all date and datetime fields
+- *Default Threshold Value*: 1
```

Comment:
i think it'd be clearer if this was 1%.  (realize the others are just "0" - maybe those could also be updated to be 0%?)